pkg/events/client_test.go

```diff
@@ -42,7 +42,7 @@ func TestListEvents(t *testing.T) {
 		require.Equal(t, expectedResponse, events)
 	})
 
-	t.Run("ListEvents succeeds to fetch Events with a startRange and endRange", func(t *testing.T) {
+	t.Run("ListEvents succeeds to fetch Events with a RangeStart and RangeEnd", func(t *testing.T) {
```

Comment:
`range_start` and `range_end`?

Reply:
ah yeah I went back and forth on that! ready for another look.